src/main/zapHomeFiles/hudtutorial/en_GB/ToggleScript.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Toggle Script</title>
+	<link href="tutorial.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+<div class="roundContainer">
+	<h1>Toggle Script</H1>
+	ZAP Scripts are a powerful way to take advantage of the ZAP engine. 
```

Comment:
`ZAP Scripts are a powerful way to extend and customise ZAP.`? (I don't think we use "ZAP engine" in other places.)